Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.xaml

```diff
@@ -0,0 +1,340 @@
+                                          ContentTransitions="{TemplateBinding ContentTransitions}"
+                                          Content="{TemplateBinding Content}"
+                                          HorizontalContentAlignment="{TemplateBinding HorizontalContentAlignment}"
+                                          Padding="{TemplateBinding Padding}"
+                                          VerticalContentAlignment="{TemplateBinding VerticalContentAlignment}"
+                                          HorizontalAlignment="Left"
+                                          RenderTransformOrigin="0.5,0.5" />
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="controls:Expander">
+        <Setter Property="Header" Value="Header" />
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="controls:Expander">
+                    <Grid>
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="ExpandedStates">
+                                <VisualState x:Name="Expanded">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="Auto" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                                <VisualState x:Name="Collapsed">
+                                    <VisualState.Setters>
+                                        <Setter Target="PART_MainContentRow.Height" Value="0" />
+                                    </VisualState.Setters>
+                                </VisualState>
+                            </VisualStateGroup>
+                        </VisualStateManager.VisualStateGroups>
+                        
+                        <Grid>
+                            <Grid.RowDefinitions>
+                                <RowDefinition Height="Auto" />
+                                <RowDefinition Height="Auto" x:Name="PART_MainContentRow" />
+                            </Grid.RowDefinitions>
+                            
+                            <ToggleButton x:Name="PART_ExpanderToggleButton" Height="40"
+                                          Style="{StaticResource HeaderToggleButtonStyle}" 
+                                          VerticalAlignment="Top" HorizontalAlignment="Stretch"
+                                          ContentTemplate="{TemplateBinding HeaderTemplate}" Content="{TemplateBinding Header}"
+                                          IsChecked="{Binding IsExpanded, Mode=TwoWay, RelativeSource={RelativeSource TemplatedParent}}" />
+                            
+                            <ContentControl Content="{TemplateBinding Content}" Grid.Row="1" HorizontalContentAlignment="Stretch" />
```

Comment:
Set HorizontalAlignment to HorizontalContentAlignment.
Replace the ContentControl with a ContentPresenter
Can remove the content binding. A ContentPresenter will bind the Content to the Content property of a ContentControl by default.